sdk/lib/_http/overrides.dart

```diff
@@ -54,7 +52,7 @@ abstract class HttpOverrides {
           findProxyFromEnvironment}) {
     HttpOverrides overrides =
         _HttpOverridesScope(createHttpClient, findProxyFromEnvironment);
-    return _asyncRunZoned<R>(body,
+    return runZoned<R>(body,
```

Comment:
Ah I see, thank you! (Originally wanted to play with Dart code review system because I may submit serious PRs in the future so want to get familiar with it).